docs/using/platform/dbaas.md

```diff
@@ -6,12 +6,14 @@
 
 The DBaaS dashboard is where you add, remove, and operate on Kubernetes and database clusters.
 
-> You must run PMM Server with a DBaaS feature flag to activate the features described here.
+## Activate a DBaaS feature
+
+**To activate the *DBaaS* feature**, select {{ icon.bars }} *PMM* --> *PMM Settings* --> *Advanced settings*.
+Then turn on the DBaaS feature by clicking on {{ icon.toggle-off }} toggle in the *Technical preview features* section of the page.
```

Comment:
The icons are defined as variables in `variables.yml`, and as far as I recall can't have dashes in the names. For example: `  times: '<i class="uil uil-times"></i>'` is then used as  `icon.times`.
The unicons icons are listed at see https://iconscout.com/unicons/explore/line. You (or I) will need to add them to make the build work.

Reply:
I added the toggle-off icon. It seems to work well.